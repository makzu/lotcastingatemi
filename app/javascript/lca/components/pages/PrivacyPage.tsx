import Typography from '@mui/material/Typography'

import BlockPaper from 'components/shared/BlockPaper'

const PrivacyPage = () => (
  <BlockPaper>
    <Typography variant="h5" gutterBottom>
      Privacy Policy and Legal Mumbo-Jumbo
    </Typography>

    <Typography variant="subtitle1">Information collection and use</Typography>
    <Typography paragraph>
      Let&apos;s cut straight to the chase: Lot-Casting Atemi saves your email
      address when you use the &apos;log in with Google&apos; link. It&apos;s
      needed for the login function to work. No other personal information is
      stored or collected. Addresses are not sent to other players.
    </Typography>
    <Typography paragraph>
      I will never use your email address for anything other than
      authentication. Period. Your personal information will never be sold or
      otherwise transferred to anyone else, except in the case where LCA changes
      hosting providers (and then only as needed to facilitate that transfer),
      or when compelled to by a court order or something.
    </Typography>

    <Typography variant="subtitle1">Cookies</Typography>
    <Typography paragraph>
      Lot-Casting Atemi uses Cookies and similar techniques to help the site
      work better. If you have Cookies disabled, portions of the site might not
      work.
    </Typography>

    <Typography variant="subtitle1">Service Providers</Typography>
    <Typography
      paragraph
      component="a"
      href="https://www.heroku.com/policy/security"
    >
      This data is stored on Heroku database servers. Heroku has its own set of
      security and privacy policies that apply to it.
    </Typography>

    <Typography variant="subtitle1">Links to Other Sites</Typography>
    <Typography paragraph>
      There are a few links to other sites in the app. Those other sites are not
      operated by me and have their own privacy policies and so on
    </Typography>

    <Typography variant="subtitle1">Changes to this Policy</Typography>
    <Typography paragraph>
      This privacy policy might update from time to time, so if you&apos;re
      concerned by these sorts of things you can periodically check this page
      for any changes.
    </Typography>

    <Typography paragraph>
      Lot-Casting Atemi is a passion project, provided to the public for free.
      It comes with no warranty or guarantee or anything.
    </Typography>

    <Typography paragraph>
      Exalted is owned by White Wolf Publishing AB and published by Onyx Path.
    </Typography>
  </BlockPaper>
)

export default PrivacyPage
