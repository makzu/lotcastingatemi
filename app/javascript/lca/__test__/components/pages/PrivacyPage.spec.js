// @flow
import PrivacyPage from '../../../components/pages/PrivacyPage.jsx'
import renderer from 'react-test-renderer'

test('PrivacyPage renders at all', () => {
  const component = renderer.create(<PrivacyPage />)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
