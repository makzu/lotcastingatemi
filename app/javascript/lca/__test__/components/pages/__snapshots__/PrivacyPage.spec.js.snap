// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrivacyPage renders at all 1`] = `
<div
  className="MuiPaper-root BlockPaper-root-1 MuiPaper-elevation1 MuiPaper-rounded"
>
  <h5
    className="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
  >
    Privacy Policy and Legal Mumbo-Jumbo
  </h5>
  <h6
    className="MuiTypography-root MuiTypography-subtitle1"
  >
    Information collection and use
  </h6>
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
  >
    Let's cut straight to the chase: Lot-Casting Atemi saves your email address when you use the 'log in with Google' link. It's needed for the login function to work. No other personal information is stored or collected. Addresses are not sent to other players.
  </p>
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
  >
    I will never use your email address for anything other than authentication. Period. Your personal information will never be sold or otherwise transferred to anyone else, except in the case where LCA changes hosting providers (and then only as needed to facilitate that transfer), or when compelled to by a court order or something.
  </p>
  <h6
    className="MuiTypography-root MuiTypography-subtitle1"
  >
    Cookies
  </h6>
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
  >
    Lot-Casting Atemi uses Cookies and similar techniques to help the site work better. If you have Cookies disabled, portions of the site might not work.
  </p>
  <h6
    className="MuiTypography-root MuiTypography-subtitle1"
  >
    Service Providers
  </h6>
  <a
    className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
    href="https://www.heroku.com/policy/security"
  >
    This data is stored on Heroku database servers. Heroku has its own set of security and privacy policies that apply to it.
  </a>
  <h6
    className="MuiTypography-root MuiTypography-subtitle1"
  >
    Links to Other Sites
  </h6>
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
  >
    There are a few links to other sites in the app. Those other sites are not operated by me and have their own privacy policies and so on
  </p>
  <h6
    className="MuiTypography-root MuiTypography-subtitle1"
  >
    Changes to this Policy
  </h6>
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
  >
    This privacy policy might update from time to time, so if you're concerned by these sorts of things you can periodically check this page for any changes.
  </p>
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
  >
    Lot-Casting Atemi is a passion project, provided to the public for free. It comes with no warranty or guarantee or anything.
  </p>
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
  >
    Exalted is owned by White Wolf Publishing AB and published by Onyx Path.
  </p>
</div>
`;
